test/OrchardCore.Benchmarks/ShapeProxyBenchmark.cs

```diff
@@ -19,6 +20,10 @@ namespace OrchardCore.Benchmark
 
     [MemoryDiagnoser]
     [ShortRunJob]
+    [SuppressMessage(
+        "Performance",
+        "CA1822:Mark members as static",
+        Justification = "BenchmarkDotNet needs all benchmark methods to be instance-level.")]
```

Comment:
I didn't add this to the whole project in the end, because these are best when they're as narrow as possible, and there's no other instance of this in the project.